README.md

```diff
@@ -21,7 +21,8 @@ portsnap fetch auto
 make -C /usr/ports/sysutils/bastille install clean
 ```
 
-**Git**
+**Git (for testing only)**
+This will break your running (config) files
```

Comment:
Is this message still valid if we merge the `.sample` PR?

Reply:
This is plan B if we don’t switch to .sample. 
Might be closed

Reply:
This is plan B if we don’t switch to .sample. 
Might be closed